bedrock/firefox/views.py

```diff
@@ -666,6 +668,20 @@ def get_template_names(self):
         return [template]
 
 
+class WhatsNewIndiaView(WhatsnewView):
```

Comment:
@alexgibson I do like this better! Was going to suggest this yesterday and failed to get around to it. I do have a couple of NITs, but nothing major.